CMakeLists.txt

```diff
@@ -169,13 +171,13 @@ if (USE_TLS)
 
     # This OPENSSL_FOUND check is to help find a cmake manually configured OpenSSL
     if (NOT OPENSSL_FOUND)
-      find_package(OpenSSL REQUIRED)
+	    include(FindOpenSSL)
     endif()
     message(STATUS "OpenSSL: " ${OPENSSL_VERSION})
 
-    add_definitions(${OPENSSL_DEFINITIONS})
-    target_include_directories(ixwebsocket PUBLIC ${OPENSSL_INCLUDE_DIR})
-    target_link_libraries(ixwebsocket ${OPENSSL_LIBRARIES})
+    #add_definitions(${OPENSSL_DEFINITIONS})
+    #target_include_directories(ixwebsocket PUBLIC ${OPENSSL_INCLUDE_DIR})
+    target_link_libraries(ixwebsocket OpenSSL::SSL)
```

Comment:
Will this include 'libcrypto' as well ?

Reply:
should it? I was able to compile link and install with just SSL for linux amd64 w/ openssl 1.1